### PR TITLE
Added Custom Action

### PR DESCRIPTION
**Pull request by greyshirtguy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -255,6 +266,16 @@ instance.prototype.action = function(action) {
 			args.push({ type: 'i', value: '0' });
 			cmd = '/track/' + opt.track + '/recarm';
 			break;
+		
+		case 'custom_action':
+			// Integer & String commandID's are sent differently...
+			if (parseInt() > 0) {
```
Comment (haakonnessjoen) [inline index.js]: This will never be true

Comment (greyshirtguy) [reply index.js]: *sigh* no of course not!....
let me see where I have screwed up and left that out

Comment (greyshirtguy) [reply index.js]: Fixed...sorry about that!
(Wow you are sharp!)

Comment (greyshirtguy) [reply index.js]: Done!

**Comment by greyshirtguy** [comment]
Thanks...btw this pull request addresses issue 1: https://github.com/bitfocus/companion-module-cockos-reaper/issues/1
You can probably close that now

**Pull request merged**